Bottom: c2257e46e90d3aed9c72f9a599bf310823603fce
Top:    4014a2ec13c90f3f28b62b03f7801894d37b238c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 07:02:46 +0100

Refresh of hid-gtk-separate-out-board-fli

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 8ec857f..c29556d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1374,7 +1374,6 @@ ghid_pcb_to_event_coords (Coord pcb_x, Coord pcb_y, int *event_x, int *event_y)
 static void
 pan_common (GHidPort *port)
 {
-  render_priv *priv = gport->render_priv;
   int event_x, event_y;
 
   /* We need to fix up the PCB coordinates corresponding to the last
@@ -1403,6 +1402,8 @@ pan_common (GHidPort *port)
 void
 ghid_pan_view_abs (Coord pcb_x, Coord pcb_y, int widget_x, int widget_y)
 {
+  render_priv *priv = gport->render_priv;
+
   gport->view_x0 = SIDE_X (pcb_x) - widget_x * gport->zoom;
   gport->view_y0 = SIDE_Y (pcb_y) - widget_y * gport->zoom;
